docs/jwst/introduction.rst

```diff
@@ -412,60 +433,39 @@ logging level designations of ``DEBUG``, ``INFO``, ``WARNING``, ``ERROR``, and
 ``CRITICAL``. Only messages at or above the specified level
 will be displayed.
 
+.. _`Configuration Files`:
 
 Configuration Files
 ===================
 
-Configuration (.cfg) files can be used to specify parameter values
-when running a pipeline or individual steps, as well as for
-specifying logging options.
-
-You can use the ``collect_pipeline_cfgs`` task to get copies of all the cfg
-files currently in use by the jwst pipeline software. The task takes a single
-argument, which is the name of the directory to which you want the cfg files
-copied. Use '.' to specify the current working directory, e.g.
-::
+Configuration files can be used to specify parameter values when running a
+pipeline or individual steps. For JWST, configuration files are retrieved from
+CRDS, just as with other reference files. If there is no match between a step,
+the input data, and CRDS, the coded defaults are used. These values can be
+overridden either by the command line options, as previously described, and by a
+local configuration file.
 
- $ collect_pipeline_cfgs .
+A configuration file should be used when there are parameters a user wishes to
+change from the default/CRDS version, but will not be changed on every run of
+the step. To create a configuration file with the default
```

Comment:
Will this work for a pipeline, or only for a step?  If I use `--save_parameters` when the class is for a pipeline, will I get the default ASDF configuration file for every step in the pipeline?

Reply:
This should only operate on the specific step/pipeline.